R/register.R

```diff
@@ -279,7 +279,9 @@ get_cpp_register_needs <- function() {
 
 check_valid_attributes <- function(decorations) {
 
-  bad_decor <- !decorations$decoration %in% c("cpp11::register", "cpp11::init")
+  bad_decor <- grepl("cpp11::", decorations$decoration) & (!decorations$decoration %in% c("cpp11::register",
```

Comment:
`startsWith()` would be slightly preferred here.

If you do want to use `grepl()` you need to anchor it to the start of the string, e.g. `grepl("^cpp11::")`

Reply:
Yep you're right! Thanks for the review